template/__hooks/__postprocess.py.jinja

```diff
@@ -0,0 +1,175 @@
+import requests
```

Comment:
This looks like a cookiecutter file. Is it still being used?

Reply:
Yes, this script is being run as a post gen script